src/subcommand/wallet/identify.rs

```diff
@@ -110,6 +110,7 @@ fn identify_from_tsv(
 #[cfg(test)]
 mod tests {
   use super::*;
+  use std::fmt::Write;
```

Comment:
Put in same item:

```suggestion
  use {super::*, std::fmt::Write};
```

Or, if possible, just import std::fmt::Write in crate root?